up-linux-streamer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "up-linux-streamer"
+rust-version.workspace = true
+version.workspace = true
+repository.workspace = true
+homepage.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+
+[dependencies]
+tokio = { version = "1.35.1", default-features = false }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "3a50104421a801d52e1d9c68979db54c013ce43d" }
```

Comment:
IMHO you should use the workspace dependency

Reply:
Agreed

Reply:
Done